apps/antalmanac/src/components/dialogs/AddSchedule.tsx

```diff
@@ -52,7 +60,14 @@ function AddScheduleDialog({ onClose, onKeyDown, ...props }: DialogProps) {
 
             <DialogContent>
                 <Box padding={1}>
-                    <TextField fullWidth label="Name" onChange={handleNameChange} value={name} />
+                    <TextField
+                        fullWidth
+                        label="Name"
+                        onChange={handleNameChange}
+                        value={name}
+                        error={!!errorMessage}
```

Comment:
Please use `Boolean` instead of `!!`. That's less explicit and quite weird to read.